internal/otgutils/print.go

```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	out.WriteString("\n\n")
 	t.Log(out.String())
 }
+
+// LogLagMetrics is displaying otg lag stats.
```

Comment:
Need to uppercase LAG due to Go style initialism rules.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done